assets/js/frontend/course-video/youtube-extension.js

```diff
@@ -0,0 +1,81 @@
+( () => {
+	// Use of Page Visibility API. Unfortunately, it isn't standardized.
+	// https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API
+	let hidden, visibilityChange;
```

Comment:
I think it's worth some tests in the different browsers to confirm if we really need that. But [it](https://caniuse.com/mdn-api_document_hidden) [seems](https://caniuse.com/pagevisibility) we don't need it. If we realize that we really need it, probably would be good to extract it making some abstraction to simplify the usage. ;)

Reply:
Tested in Chrome, Firefox, Opera & Edge. Everything works.

Reply:
Fixed it here: https://github.com/Automattic/sensei/pull/4546/commits/26cf9fca7c310770325294eb77147e8a1cc124e5